### ISSUE TITLE
[IntelliJ][ReportedByUser] Uncaught Exception  java.lang.IndexOutOfBoundsException: Index 0 out of bounds for length 0

### ISSUE THREAD

**Issue by verejanvasile** [microsoft/azure-tools-for-java#6846]
IntelliJ build version: 2022.1.3 IU-221.5921.22
OS: Windows 11
JDK: JetBrains s.r.o. 11.0.15
Plugin version: 3.67.0-2022.1
Additional Info: None
Parent component: 
```
com.intellij.openapi.wm.impl.IdeRootPane[,7,7,1537x816,layout=com.intellij.openapi.wm.impl.IdeRootPane$MyRootLayout,alignmentX=0.0,alignmentY=0.0,border=,flags=449,maximumSize=,minimumSize=,preferredSize=]
```
Error message: 
```
IdeaLoggingEvent[message=null, throwable=java.lang.IndexOutOfBoundsException: Index 0 out of bounds for length 0
	at java.base/jdk.internal.util.Preconditions.outOfBounds(Preconditions.java:64)
	at com.microsoft.azure.toolkit.intellij.legacy.appservice.AppServiceInfoBasicPanel.init(AppServiceInfoBasicPanel.java:64)
	at com.microsoft.azure.toolkit.intellij.legacy.appservice.AppServiceInfoBasicPanel.<init>(AppServiceInfoBasicPanel.java:60)
	at com.microsoft.azure.toolkit.intellij.legacy.webapp.WebAppCreationDialog.createUIComponents(WebAppCreationDialog.java:62)
	at com.microsoft.azure.toolkit.intellij.legacy.webapp.WebAppCreationDialog.$$$setupUI$$$(WebAppCreationDialog.java)
	at com.microsoft.azure.toolkit.intellij.legacy.webapp.WebAppCreationDialog.<init>(WebAppCreationDialog.java:28)
	at com.microsoft.azure.toolkit.intellij.legacy.webapp.runner.webappconfig.slimui.WebAppComboBox.createResource

**Comment by wangmingliang-ms** [comment]
This bug should be fixed by commit fabd7859, this fix will be included in the next stable release. 
thanks for your support.

**Comment by wangmingliang-ms** [comment]
close as it has been fixed since v3.69.0, you can upgrade to the latest and try. Feel free to reopen it when you encounter this error again.
